wheel/stubtest.allowlist

```diff
@@ -29,6 +29,9 @@ chia_rs\.PrivateKey\.__init__
 # TODO: expects *args
 chia_rs\.RequestPeers\.__init__
 
+# TODO: expects *args
```

Comment:
I don't think this comment is right. This is an empty struct, that's why it has to be exempt (afaict)

Reply:
I just copied the existing stubtest comments on similar empty structs - I think "expects *args" is the warning that's created by being an empty struct for some reason. @altendky probably knows more about this

Reply:
The full error is `chia_rs.RequestCostInfo.__init__ is inconsistent, stub does not have *args argument "args"`.  The Rust form of `RequestCostInfo` matches that of `RequestPeers` above with a similar comment.  It's a todo because it would be nice to figure out what is going on at some point.  I'm ok with continuing this ignore pattern with this addition.

Reply:
this being a `TODO`-comment makes it look like there's some improvement that can be made here, that we're punting on. It's not clear to me what that improvement would be though (at least not from this comment). I suppose we could improve the stub generation to support empty structs.